apps/judicial-system/web/src/utils/hooks/useSections/index.ts

```diff
@@ -1122,94 +1122,85 @@ const useSections = (
         isInvestigationCase(type) &&
         Boolean(parentCase) &&
         !isCompletedCase(state),
-      children:
-        user?.institution?.type !== InstitutionType.PROSECUTORS_OFFICE
-          ? []
-          : [
-              {
-                name: capitalize(
-                  formatMessage(core.defendant, { suffix: 'i' }),
-                ),
-                isActive: isActive(
-                  constants.INVESTIGATION_CASE_DEFENDANT_ROUTE,
-                ),
-                href: `${constants.INVESTIGATION_CASE_DEFENDANT_ROUTE}/${id}`,
-              },
-              {
-                name: formatMessage(
-                  sections.investigationCaseExtensionSection
-                    .hearingArrangements,
-                ),
-                isActive: isActive(
+      children: !(
+        user?.institution?.type &&
+        prosecutorsOfficeTypes.includes(user.institution.type)
```

Comment:
Could also use type check helper mentioned above here

Reply:
Great comments 🙏

Please consider accepting PRs like this one that only require such minor adjustments so that we can speed up the merge process 🫣.